pallets/pallet-nft/src/tests.rs

```diff
@@ -0,0 +1,210 @@
+#![cfg(test)]
+
+use crate::{mock::*, *};
+use frame_support::{assert_noop, assert_ok};
+
+/// Set up an Nft to each storage which is easier to test burn, transfer and audit functionality.
+fn set_up_nfts() {
+	Nfts::<Runtime>::insert(
+		1u32,
```

Comment:
maybe use the "next_nft_id" to get this number, so internally the counter is incremented.